src/sentry/sentry_metrics/querying/samples_list.py

```diff
@@ -287,13 +301,23 @@ def execute(self, offset, limit):
         return result
 
     def get_span_keys(self, offset: int, limit: int) -> list[tuple[str, str, str]]:
+        column = self.mri_to_column(self.mri)
+
+        query_parts = []
+        if self.query:
+            query_parts.append(self.query)
+        if self.min is not None:
+            query_parts.append(f"{column}:>={self.min}")
+        if self.max is not None:
+            query_parts.append(f"{column}:<={self.max}")
```

Comment:
nit: should we move this to somewhere that can be shared?

🤔 also is there any casue where `self.query` could accidentally bleed into the other query terms? Should we wrap it in `()` just to be safe?

Reply:
I've moved all the implicit conditions into the additional conditions so this is no longer an issue.